src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana-details-panel/grafana-details-panel.component.html

```diff
@@ -0,0 +1,21 @@
+<!-- Embed details dashboard -->
+<!-- TODO: display loading spinner as long as the images haven't loaded -->
+<cd-info-panel>
+  <ng-container i18n *ngIf="grafanaExist">
```

Comment:
This template is exactly the same as the other grafana HTML. Could this suggest that you may re-use some code or join components?

Reply:
Ack, will give it a try.